### ISSUE TITLE
Application complete page needs to show message if notifications not selected

### ISSUE THREAD

**Issue by lizziebruce** [alphagov/verify-local-patterns#675]
If user hasn’t opted in for a confirmation message or renewal reminder this message should display above the horizontal line:

> “You have not signed up to receive a confirmation email. Sign up for confirmation and progress alerts.” 

![screen shot 2017-09-12 at 10 55 18](https://user-images.githubusercontent.com/27814324/30322795-96914bea-97b2-11e7-83a4-2bb708700af4.png)

**Comment by colin2678** [comment]
We cannot see how this can be achieved. 
Once we have issued a confirmation number the form will have been submitted to Unicard. We would need to reopen the application in order to update these records, which would require a new reference number.

**Comment by lizziebruce** [comment]
@colin2678 Hi Colin, I went through the Bucks journey without selecting confirmation or renewals notifications. When a user does that, ideally this copy should show up on the application complete page:

> You have not signed up to receive a confirmation email. Sign up for confirmation and progress alerts.

It should show above the horizontal line above the What happens next section – please compare with Argleton screenshot in first comment above. Thanks.

![30321814-5198a784-97af-11e7-92e8-04156e891aff](https://user-images.githubusercontent.com/27814324/30477494-494b3dc8-9a05-11e7-9a81-4bf892a58464.png)

**Comment by colin2678** [comment]
Hi Lizzie. You misunderstand I know where the text is. I just don't see how we can do this. Once a reference number has been issued, the form will have been submitted and the data added to Unicard. In order to change these details we would need to submit a new form to do so. I'm happy to talk this through with you if you would prefer.

**Comment by lizziebruce** [comment]
@colin2678 Hi Colin, I just wanted to make sure my comment had been expressed clearly as I couldn't see how your comment related to it. It sounds like this is a service design issue so I need to pass that on to @sanjaypoyzer Thanks both

**Comment by colin2678** [comment]
I agree this is a service pattern issue, and I think not something we are likely able to achieve using our forms system.

**Comment by suryanaga** [comment]
@colin2678 Hi Colin, if someone wanted to get in touch with the council about their application, what information would they need?

**Comment by colin2678** [comment]
We could identify the application using the reference number or the date and time of the application.

**Comment by suryanaga** [comment]
Ah sorry, I misunderstood what was being talked about here – This issue isn't anything to do with reference numbers, but the message that confirms to the user whether or not they have signed up to notifications.

**Comment by colin2678** [comment]
We can certainly add a message, but not the ability to change the users preferences

**Comment by suryanaga** [comment]
Ah ok, let's add the message then for confirmation and not have the link in this instance.

**Comment by BeckiBorastonBucks** [comment]
The message "You have not signed up to receive a confirmation email" implies that it should only be shown if the user has not selected 'Email'. What about SMS?

**Comment by lizziebruce** [comment]
@BeckiBorastonBucks 

The variables should be:

[User selects email only]
You'll receive a confirmation email in the next 10 minutes.

[User selects SMS only]
You'll receive a confirmation SMS message in the next 10 minutes.

[User selects both]
You'll receive a confirmation email and SMS message in the next 10 minutes.

[User selects neither]
You have not signed up to receive a confirmation SMS or email. Sign up for confirmation and progress alerts.

**Comment by BeckiBorastonBucks** [comment]
Can we say 'text' instead of 'SMS'?

**Comment by EUzkuraityte** [comment]
@BeckiBorastonBucks Yes, most definitely, it should say 'text message'
